src/Symfony/Bundle/WebProfilerBundle/Tests/DependencyInjection/WebProfilerExtensionTest.php

```diff
@@ -100,6 +100,8 @@ public function testDefaultConfig($debug)
     }
 
     /**
+     * @group legacy
```

Comment:
same here about splitting the testing of legacy features. This avoids hiding other unexpected deprecations.